lib/check_params.h

```diff
@@ -723,13 +723,9 @@ void printQudaMultigridParam(QudaMultigridParam *param) {
     }
 
 #ifdef INIT_PARAM
-    if (i<QUDA_MAX_MG_LEVEL) {
-          P(n_vec[i], INVALID_INT);
-    }
+    if (i < QUDA_MAX_MG_LEVEL) { P(n_vec[i], INVALID_INT); }
```

Comment:
@cpviolator not that I expect this to be controversial at all, but can you please remove this change? It's unrelated to the PR, and since it's a PR against release we may as well keep everything on a narrow path.